feat(resource): New lacework_resource_group_gcp (#156)
lacework/terraform-provider-lacework: website/docs/r/resource_group_gcp.html.markdown

---
subcategory: "Resource Groups"
layout: "lacework"
page_title: "Lacework: lacework_resource_group_gcp"
description: |-
Create and manage GCP Resource Groups
---

# lacework\_resource\_group\_gcp

Use this resource to create an GCP Resource Group in order to categorize Lacework-identifiable assets.
For more information, see the [Resource Groups documentation](https://support.lacework.com/hc/en-us/articles/360041727354-Resource-Groups).

## Example Usage

```hcl
resource "lacework_resource_group_gcp" "example" {

>   name        = "My GCP Resource Group"
>   description = "This groups a subset of Gcp Projects"
>   accounts    = ["123456789", "234567891"]

Author: ```suggestion
  name         = "My GCP Resource Group"
  description  = "This groups a subset of Gcp Projects"
  projects     = ["123456789", "234567891"]
  organization = "MyGcpOrg"
```

---
subcategory: "Resource Groups"
layout: "lacework"
page_title: "Lacework: lacework_resource_group_gcp"
description: |-
Create and manage GCP Resource Groups
---

# lacework\_resource\_group\_gcp

> Use this resource to create an GCP Resource Group in order to categorize Lacework-identifiable assets.

Reviewer: ```suggestion
Use this resource to create a GCP Resource Group in order to categorize Lacework-identifiable assets.
```

For more information, see the [Resource Groups documentation](https://support.lacework.com/hc/en-us/articles/360041727354-Resource-Groups).

## Example Usage

```hcl
resource "lacework_resource_group_gcp" "example" {
  name         = "My GCP Resource Group"
  description  = "This groups a subset of Gcp Projects"

>   projects     = ["123456789", "234567891"]

Reviewer: these don't look like real project ids, lets use these:
```suggestion
  projects     =  ["project-1", "project-2", "project-3"]
```

>   organization = "MyGcpOrg"

Reviewer: ```suggestion
  organization = "MyGcpOrgID"
```

}
```

## Argument Reference

The following arguments are supported:

* `name` - (Required) The resource group name.

> * `accounts` - (Required) The list of GCP projects to include in the resource group.

Reviewer: copy paste? 🦢 ❤️ 😂
```suggestion
* `projects` - (Required) The list of GCP project IDs to include in the resource group.
```

> * `organization` - (Required) The GCP organization.

Reviewer: ```suggestion
* `organization` - (Required) The GCP organization ID.
```
